website/gatsby-node.js

```diff
@@ -133,3 +137,19 @@ exports.createPages = ({ graphql, boundActionCreators }/* : NodeParams */)/* : P
     );
   });
 };
+
+exports.onCreatePage = async ({ page, boundActionCreators }) => {
+  const { createPage } = boundActionCreators;
+
+  return new Promise((resolve, reject) => {
```

Comment:
kewl! Does this still work well with SSR?

Reply:
Does it need to be a promise?

Reply:
This is the recommended way to do this from gatsby's docs. I don't think it should hit SSR, as this is run when gatsby is generating the site's data.